### DISCUSSION TITLE
space-x should update variable to just work in rtl languages

### DISCUSSION THREAD

**Discussion by scottromack** [tailwindlabs/tailwindcss#8986] [Ideas]
**What version of Tailwind CSS are you using?**

Tailwind v3.1.2

**What version of Node.js are you using?**

v14.17.3

**What browser are you using?**

Mobile first and only modern browsers

**What operating system are you using?**

macOS

**Reproduction URL**

https://play.tailwindcss.com/MWbXRd47S6

**Describe your issue**

in rtl --tw-space-x-reverse: 0; should be set to 1

Currently, in rtl the margin is to the left instead of switching to the right via the variable noted above.

**Comment by abdelhak002** [comment]
@ShaggyDude 
Add `space-x-reverse` to set the `--tw-space-x-reverse` to 1. 
See Docs [https://tailwindcss.com/docs/space#reversing-children-order](https://tailwindcss.com/docs/space#reversing-children-order)
